build.gradle.kts

```
@@ -60,6 +60,7 @@ buildscript {
     project.extra.set("artifactoryUrl", artifactoryUrl)
     project.extra.set("artifactoryUsername", artifactoryUsername)
     project.extra.set("artifactoryPassword", artifactoryPassword)
+    project.extra.set("sdkToolkitTestsDir", projectDir.absolutePath.removeSuffix("arcgis-maps-sdk-kotlin-toolkit") + "/kotlin/sdk-toolkit-tests")
```

```suggestion
    project.extra.set("internalToolkitTestsDir", projectDir.absolutePath.removeSuffix("arcgis-maps-sdk-kotlin-toolkit") + "/kotlin/internal-toolkit-tests")
```
Perhaps `internalToolkitTestsDir` is a more appropriate name?